montecarlo/evac_mcarlo.py

```diff
@@ -105,7 +121,14 @@ def _make_pre_evacuation(self,room,type_sec):# {{{
             pe=self.conf['pre_evac']
         else:
             pe=self.conf['pre_evac_fire_origin']
-        return self._get_alarming_time() + round(lognorm(s=1, loc=pe['mean'], scale=pe['sd']).rvs(), 2)
+
+        if pe['mean'] and pe['sd']:
+            # if distribution parameters are given
+            return self._get_alarming_time() + round(lognormal(mean=pe['mean'], sigma=pe['sd']), 2)
+        elif pe['1st'] and pe['99th']:
```

Comment:
if for example pe['mean'] and pe['1st'] are empty then nothing will be returned. is it ok?

Reply:
As in previous comment - I'll add some lines to handle those situations.